modules/apps/account/account-service/src/main/java/com/liferay/account/internal/model/listener/AccountGroupRelModelListener.java

```diff
@@ -36,14 +36,14 @@
 	public void onAfterCreate(AccountGroupRel accountGroupRel)
 		throws ModelListenerException {
 
-		_reindexAccountEntry(accountGroupRel.getAccountEntryId());
+		_reindexAccountEntry(accountGroupRel.getClassPK());
```

Comment:
You'll need to check the `classNameId` to be sure you're dealing with an AccountEntry, otherwise you may pass an invalid PK.